### PR TITLE
[SOI] Implemented Bonds of Moonsilver

### PR DESCRIPTION
**Pull request by halljared**
I tested this with a bunch of new werewolves, and with a Delver of Secrets as a representative from the old school flip cards.

### CODE REVIEW

File: Mage/src/main/java/mage/abilities/effects/common/CantAttackBlockTransformAttachedEffect.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+/**
+ *
+ * @author Jared
+ */
+public class CantAttackBlockTransformAttachedEffect extends RestrictionEffect{
+    
+    public CantAttackBlockTransformAttachedEffect() {
+        super(Duration.WhileOnBattlefield);
+        staticText = "Enchanted creature can't attack or block, and its activated abilities can't be activated";
```
Comment (drmDev) [inline Mage/src/main/java/mage/abilities/effects/common/CantAttackBlockTransformAttachedEffect.java]: The text should be changed to reflect it not allowing transform.

File: Mage.Sets/src/mage/sets/shadowsoverinnistrad/BoundByMoonsilver.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        // Enchanted creature can't attack, block, or transform.
+        // Sacrifice another permanent: Attach Bound by Moonsilver to target creature. Activate this ability only any time you could cast a sorcery and only once each turn.
+        // Enchant nonland permanent
+        TargetPermanent auraTarget = new TargetCreaturePermanent();
+        this.getSpellAbility().addTarget(auraTarget);
+        this.getSpellAbility().addEffect(new AttachEffect(Outcome.Detriment));
+        Ability ability = new EnchantAbility(auraTarget.getTargetName());
+        this.addAbility(ability);
+
+        // Enchanted permanent can't attack or block, and its activated abilities can't be actiated.
```
Comment (drmDev) [inline Mage.Sets/src/mage/sets/shadowsoverinnistrad/BoundByMoonsilver.java]: Comment should be changed to reflect cannot transform, not activated abilities.

**Comment by drmDev** [comment]
As far as the card implementation goes, looks good for the most part. I included a couple comments on the text not matching the ability (looks like it was borrowed from cannot attack, block, or activate abilities). Functionally should be good though.

Also the pull requests can be a little cleaner in the future by merging the remote branch and committing and pushing that to your local repo first. Then you commit your new file changes and push that for the pull request - that way the pull request only has the commit of the file you changed. For instance, this pull request includes Apothecary Geist as well as two merges, and the Bound by Moonsilver file.

**Comment by halljared** [comment]
Hey I followed a guide about how to rebase my commits to make the history cleaner. I ended up deleting the Apothecary Geist stuff as that was an oversight on my part in the first place. I fixed the strings you mentioned and also my author name in the files that are new. Please give this another look if you could, as its a shiny new pull request (albeit with the same stuff as before, all squashed together).

**Pull request merged**